playbooks/group_vars/all/global.yml

```diff
@@ -16,11 +16,11 @@ force_repo_clone: true
 contra_env_setup_dir: "{{ ansible_env.HOME }}/.contra-env-setup"
 
 # Docker KVM version
-dkvm_version: v0.7.0
+dkvm_version: v0.10.0
 
 ## minishift values
 # minishift version
-minishift_version: v1.12.0
+minishift_version: v1.17.0
```

Comment:
We can't support this as the default until we update the Custom CentOS ISO or we need to change the default CentOS ISO from minishift. I vote for the former since that will break others.